### DISCUSSION TITLE
Mania playfield is being overly lightened in replay when rewinding for osu standart skin

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19118] [Q&A] [answered]
I dont think this is intentional

https://user-images.githubusercontent.com/86934170/178926551-e88374d3-f062-4cd9-963b-ae6476e1f246.mp4

**Comment by peppy** [comment]
Was sure this was already tracked somewhere, but maybe not.

**Comment by Loreos7** [comment, marked as answer]
https://github.com/ppy/osu/issues/9117
